package.json

```diff
@@ -4,7 +4,9 @@
   "description": "Alex's GraphQL project",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
```

Comment:
this test isnt doing anything. If code isnt doing anything it should be pulled out

Reply:
Done